app/views/publish_mailer/publish_email.text.erb

```diff
@@ -0,0 +1,36 @@
+<%= @edition.document_type.label %> ‘<%= @edition.title %>’
+
+<% if @edition.number > 1 %>
+  <%= t("publish_mailer.publish_email.details.update",
+    time: @status.created_at.strftime("%l:%M%P"),
+    date: @status.created_at.strftime("%d %B %Y"),
+    user: @status.created_by.name,
+  ) %>
```

Comment:
It looks a bit like this would indent the text being echoed here. Does that cause any oddities with either the HTML or text view in the Notify email?

It might need to use the erb whitespace suppression features.

Reply:
Good thought - I'd neglected to check the text variant. Turns out it's mostly OK, apart from a couple of excess newlines. I've added a commit to sort this out a bit, but it's still not perfect (there will be a blank line if there's no 2i warning), and I wonder if it's better to just not worry too much and write templates in the normal HTML style?

Reply:
I think we should try make best efforts with it as it'll be so hard to quantify whether or not people are seeing the text ones and we'd want them to not look broken.

Thanks for sorting those bits out.

It could be easier to have a grip on the whitespace by writing them in a less HTML style e.g.

```
<%= 
if @edition.number > 1
  if @edition.major?
    t("publish_mailer.publish_email.change_note") + "\n" + @edition.change_note
 else
    t("publish_mailer.publish_email.minor_update")
end
%>

<%= t("publish_mailer.publish_email.2i_warning") if @edition.published_but_needs_2i? %>
```

Reply:
I had a go at that one, but unfortunately it recreates the problem of an extra blank line if there's no 2i warning - it's tempting to have a second view to cope with this case, but I think they're too similar to justify that.